pkgs/applications/misc/dwdiff/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, stdenv
+, fetchurl
+, gettext
+, pkg-config
+, icu
+}:
+
+stdenv.mkDerivation rec {
+  pname = "dwdiff";
+  version = "2.1.4";
+
+  src = fetchurl {
+    url = "https://os.ghalkes.nl/dist/${pname}-${version}.tar.bz2";
+    sha256 = "sha256-3xb+xE3LRn1lpCRqQ2KPk3QZlsF3PpMLkMbd4i3Vjgo=";
+  };
+
+  nativeBuildInputs = [ pkg-config ];
+
+  buildInputs = [
+    gettext
+    icu
+  ];
+
+  meta = with lib; {
+    description = "A front-end for the diff program that operates at the word level instead of the line level";
```

Comment:
```suggestion
    description = "Front-end for the wdiff program that operates at the word level instead of the line level";
```

Reply:
That doesn't make sense, [wdiff](https://www.gnu.org/software/wdiff/) is a separate front-end for diff that operates at the word level